docs/creating_integration_manifest.md

```diff
@@ -28,6 +28,7 @@ Or a minimal example that you can copy into your project:
 {
   "domain": "your_domain_name",
   "name": "Your Integration",
+  "version": "0.0.1",
```

Comment:
Thanks for the suggestion @tomasmcguinness 👍 

However, this documentation is aimed at building an integration for Home Assistant, not a custom integration.

Built-in integration inherits the version number from the Home Assistant core itself; adding this version key will actually cause CI failures; thus, this part is left out of this example.

../Frenck